x-pack/plugins/observability/public/hooks/slo/use_fetch_active_alerts.ts

```diff
@@ -11,17 +11,13 @@ import { BASE_RAC_ALERTS_API_PATH } from '@kbn/rule-registry-plugin/common';
 import { useKibana } from '../../utils/kibana_react';
 import { sloKeys } from './query_key_factory';
 
-type SloId = string;
+type SloIdAndInstanceId = [string, string];
 
 interface Params {
-  sloIds: SloId[];
+  sloIdsAndInstanceIds: SloIdAndInstanceId[];
 }
 
-export interface ActiveAlerts {
-  count: number;
-}
-
-type ActiveAlertsMap = Record<SloId, ActiveAlerts>;
+type ActiveAlertsMap = Record<string, number>;
```

Comment:
I'm ok with this typing for the time being, but I'd like to get a chance at refactoring it during feature freeze or in 8.12:
The `string` in Record<string, number>` is not straightforward, if you don't know that it's the `sloid|instanceId` bucket key.

Reply:
Should I change it to something like this?

```typescript
class ActiveAlerts {
  private data: Set<string, number> = new Set();

  set(slo: SLO, value: number) {
    this.data.set(`${sli.id}|${slo.instanceId ?? ALL_VALUE}`, value);
  }

  get(slo: SLO) {
    return this.data.get(`${sli.id}|${slo.instanceId ?? ALL_VALUE}`);
  }

  has(slo: SLO) {
    return this.data.has(`${sli.id}|${slo.instanceId ?? ALL_VALUE}`);
  }

}
```